spec/lib/healthcheck/government_data_check_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+RSpec.describe Healthcheck::GovernmentDataCheck do
+  describe "#status" do
+    context "when government data is unavailable" do
+      it "returns critical" do
+        gov_data_check = described_class.new
+        expect(gov_data_check.status).to be(:critical)
+      end
+    end
+
+    context "when government data is set to expire" do
+      it "gives a warning" do
+        populate_default_government_bulk_data
+        gov_data_check = described_class.new
+        travel_to(7.hours.from_now) do
+          expect(gov_data_check.status).to be(:warning)
+        end
+      end
+    end
+
+    context "when everything is fine" do
+      it "returns ok" do
+        populate_default_government_bulk_data
+        gov_data_check = described_class.new
+        expect(gov_data_check.status).to be(:ok)
+      end
+    end
+  end
+
+  describe "#details" do
```

Comment:
Looks like this needs renaming for message

Reply:
Should have spotted this, sorry. Fixed.